Makefile

```diff
@@ -185,12 +195,12 @@ LIBC_TOP_HALF_ALL_SOURCES = \
     $(shell find $(LIBC_TOP_HALF_SOURCES) -name \*.c)
 
 # Set the target.
-CFLAGS = $(WASM_CFLAGS) --target=$(TARGET_TRIPLE)
+override CFLAGS += --target=$(TARGET_TRIPLE)
```

Comment:
Could we avoid all those ugly overrides by instead doing something like:

```
CFLAGS = $(EXTRA_CFLAGS) --target=$(TARGET_TRIPLE)
```

I believe `EXTRA_CFLAGS` is a relatively standard way to inject extra `CFLAGS`.

Reply:
Yeah sure!